spec/features/orders/creation_spec.rb

```diff
@@ -29,30 +29,37 @@
     end
 
     it 'I can create a new order' do
-      name = "Bert"
-      address = "123 Sesame St."
-      city = "NYC"
-      state = "New York"
-      zip = 10001
+      user = User.create(name: "David", address: "123 Test St", city: "Denver", state: "CO", zip: "80204", email: "123@example.com", password: "password", role: 1)
+      visit '/login'
 
-      fill_in :name, with: name
-      fill_in :address, with: address
-      fill_in :city, with: city
-      fill_in :state, with: state
-      fill_in :zip, with: zip
+      fill_in :email, with: user.email
+      fill_in :password, with: user.password
+      click_on 'Login'
```

Comment:
Seems like a good spot to use a stubs. It's faster(though not to us) to just tell the program `:current_user = user` whenever we have to login